CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Replaceable HTML values for hints
+
+### Added
+
```

Comment:
```suggestion
```

Reply:
Its for after the release happens... hopefully. Or is this one coming forward a release?

Reply:
It's been bumped